spec/locale-sensitive-functions.html

```diff
@@ -267,32 +251,26 @@
       </p>
 
       <emu-alg>
-        1. Let _A_ be ToObject(*this* value).
-        2. ReturnIfAbrupt(_A_).
-        3. Let _len_ be ToLength(Get(_A_, *"length"*)).
-        4. ReturnIfAbrupt(_len_).
-        5. Let _separator_ be the String value for the list-separator String appropriate for the host environment’s current locale (this is derived in an implementation-defined way).
-        6. If _len_ is zero, return the empty String.
-        7. Let _firstElement_ be Get(_A_, *"0"*).
-        8. ReturnIfAbrupt(_firstElement_).
-        9. If _firstElement_ is *undefined* or *null*, then
+        1. Let _A_ be ? ToObject(*this* value).
+        1. Let _len_ be ? ToLength(Get(_A_, `"length"`)).
+        1. Let _separator_ be the String value for the list-separator String appropriate for the host environment’s current locale (this is derived in an implementation-defined way).
+        1. If _len_ is zero, return the empty String.
+        1. Let _firstElement_ be ? Get(_A_, *"0"*).
+        1. If _firstElement_ is *undefined* or *null*, then
           1. Let _R_ be an empty String.
-        10. Else
-          1. Let _R_ be ToString(Invoke(_firstElement_, *"toLocaleString"*, «*locales*, *options*»)).
-          1. ReturnIfAbrupt(_R_).
-        11. Let _k_ be 1.
-        12. Repeat, while _k_ < _len_
+        1. Else
+          1. Let _R_ be ? ToString(Invoke(_firstElement_, *"toLocaleString"*, «*locales*, *options*»)).
+        1. Let _k_ be 1.
+        1. Repeat, while _k_ < _len_
           1. Let _S_ be a String value produced by concatenating _R_ and _separator_.
-          1. Let _nextElement_ be Get(_A_, ToString(_k_)).
-          1. ReturnIfAbrupt(_nextElement_).
+          1. Let _nextElement_ be ? Get(_A_, ToString(_k_)).
           1. If _nextElement_ is *undefined* or *null*, then
             1. Let _R_ be the empty String.
-          1. Else
-            1. Let _R_ be ToString(Invoke(_nextElement_, *"toLocaleString"*, «*locales*, *options*»)).
-            1. ReturnIfAbrupt(_R_).
+          1. Else,
+            1. Let _R_ be ? ToString(Invoke(_nextElement_, *"toLocaleString"*, «*locales*, *options*»)).
           1. Let _R_ be a String value produced by concatenating _S_ and _R_.
           1. Increase _k_ by 1.
-        13. Return _R_.
+        1. Return _R_.
       </emu-alg>
 
       <emu-note>
```

Comment:
Just revert any incorrectly changed formatting (so I don't have to mark them all)